parse.go

```diff
@@ -83,6 +101,26 @@ func NewParserFromFile(path string, frameChannel chan *frame.Frame) (Parser, err
 	return p, err
 }
 
+// NewParserFromFile initializes and returns a new dicom Parser from a file path with the force parameter as true
+func NewParserFromFileForce(path string, frameChannel chan *frame.Frame) (Parser, error) {
+	file, err := os.Open(path)
```

Comment:
Definitely, I think that makes a lot of sense. I'll implement this

Reply:
@suyashkumar I realized something while making this change that I should've a long time ago. The reason I thought we needed 2 separate NewParser functions was because inside of `NewParser`,`parseFileHeader()` is called which is where the force flag is relevant. An easy way around this and to get rid of the dual parser functions without moving `parseFileHeader()` to the `Parse` function like we already discussed, however, would be to move p.parseFileHeader() to the parent function that calls `NewParser` (e.g. move it to `NewParserFromFile` and `NewParserFromBytes`). This would also allow for more of `NewParserFromFileForce` and `NewParserFromFile` to be shared in the helper function `newParserFromFileInternal `. The major downside I can see would be that someone couldn't call NewParser() directly without manually parsing the header, but it seems like NewParser is already a helper function and isn't intended to be called by a user?

For now, I'll go ahead and implement it as we discussed previously, but if you prefer this option I can go ahead and make the changes.